policy-controller/grpc/src/outbound.rs

```diff
@@ -407,6 +421,78 @@ fn fallback(original_dst: SocketAddr) -> outbound::OutboundPolicy {
     }
 }
 
+fn undefined_port(target: ResourceTarget) -> outbound::OutboundPolicy {
+    let metadata = Some(Metadata {
+        kind: Some(metadata::Kind::Resource(Resource {
+            group: target.kind.group().to_string(),
+            kind: target.kind.kind().to_string(),
+            name: target.name,
+            namespace: target.namespace,
+            section: String::default(),
+            port: target.port.get() as u32,
+        })),
+    });
+
+    // Since we set a Forbidden filter, this backend will never be used.
+    // However, it is required to have a backend in the OpaqueRoute in order
+    // for the config to be valid. Therefore, we use an arbitrary dummy backend.
+    let backend = outbound::Backend {
+        metadata: metadata.clone(),
+        queue: Some(default_queue_config()),
+        kind: Some(outbound::backend::Kind::Forward(
+            destination::WeightedAddr {
+                addr: Some(SocketAddr::new(IpAddr::V4(Ipv4Addr::new(1, 1, 1, 1)), 1).into()),
```

Comment:
TIOLI: `1.1.1.1` is cloudflare's DNS, which might be misleading. Something like `192.0.2.1` which is reserved for documentation/examples might be more suitable.